hack/local-cmo.sh

```diff
@@ -1,12 +1,36 @@
 #!/usr/bin/env bash
 set -eu -o pipefail
 
+cmo_login='true'
+
 # Script will:
 # 1. Patch CVO to exclude CMO from being managed
 # 2. Scale in-cluster CMO to 0
 # 3. Try to run CMO locally (CMO needs to be built before starting)
 
 
+echo "script starting"
+
+while test $# -gt 0; do
+  case "$1" in
+    -h|--help)
+      echo "local_cmo [options]"
+      echo " "
+      echo "options:"
+      echo "-h, --help                show brief help"
+      echo "-n, --no-cmo-login        run script as kube:admin"
+      exit 0
+      ;;
+    -n|--no-cmo-login)
+      cmo_login='false'
+      shift
+      ;;
+    *)
+      break
+      ;;
+  esac
+done
+
```

Comment:
Could we please move this to a parse_args function? The rule of thumb for maintainable scripts is to keep as much as possible in a function so that there aren't any adhoc execution and the flow is predictable.